<launch>
  <!-- The argument 5 is to notify the code that it is in testing mode for the robot Explorer -->
  <group ns = "test_robot_1">
    <node pkg="mdis_state_machine" type="team_scheduler" name="team_scheduler" args="6 test_robot_2 test_robot_3" output="screen"/>
  </group>

  <!-- The argument 6 is to notify the code that it is in testing mode for the robot Relay -->
  <group ns = "test_robot_2">
    <node pkg="mdis_state_machine" type="team_scheduler" name="team_scheduler" args="5 data_center test_robot_1" output="screen"/>
  </group>

  <!-- The argument 6 is to notify the code that it is in testing mode for the robot Relay -->
  <group ns = "data_center">
    <node pkg="mdis_state_machine" type="team_scheduler" name="team_scheduler" args="7 dummy_parent dummy_child" output="screen"/>
  </group>
</launch>
